zapier/introduction/zapier-task-history/zapier-filtering-task-history.md

```diff
@@ -0,0 +1,53 @@
+---
+author: kapnobatai136
+
+category: must-know
+
+aspects:
+  - introduction
+
+type: normal
+
+---
+
+# Filtering Task History
+
+---
+## Content
+
+Now that you know how to rename your zaps and how to access your task history, it is time to learn how to filter it.
+
+All the magic happens at the top of the list:
+
+![task-history-filters](https://img.enkipro.com/44d77a1e0017bc643a9e52d60be36998.png)
+
+Start from left to right, we will now explain what each option does:
+- the select button
+    - you can individually select an action or select multiple actions
+    - when selected, you can play, delete, or download them
+    - useful for retrying a certain action
+- the zap specifier
+    - in the picture you can see the `'All Zaps'` button
+    - using this, you can specify which zaps you are interested in
+    - this is where renaming zaps comes in useful
+- the status specifier
+    - in the picture, you can see the `'Everything'` button; this specifies which statuses you are interested in
+    - use this to find out which actions threw an error or which are still waiting
+- the date specifier
+    - choose a date as a constraint
+
+Lastly, you've got the refresh button (meant to refresh the actions seen in the task history), and the `'Clear Filters'` button.
+
+Using these together you will be able to debug any zaps.
```

Comment:
I think this is an overstatement. This is mainly their main debugging tool but it doesn't mean it's perfect.

```suggestion
These features should provide you with a straightforward debugging experience.
```